docs/RENDER_SETUP.md

```
@@ -0,0 +1,152 @@
+**Complexity**: Low (~1 hour total). The backend is fully stateless — Supabase owns all
+persistent data. Nothing needs to be migrated except the runtime environment.
+
+**Steps**:
+
+1. Spin up a [Hetzner CX22](https://www.hetzner.com/cloud) VPS (€4/month, 4 GB RAM,
+   Ubuntu 24.04)
+2. Install [Coolify](https://coolify.io) (one command, ~20 min)
+3. Connect the same GitHub repo in Coolify — no code changes needed
+4. Copy env vars from Render Dashboard → Coolify
+5. Transfer ML model files to VPS:
+   ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)